config/webpack.config.dev.js

```diff
@@ -89,6 +89,10 @@ var loaders = [
       {
         test: /\.svg$/,
         loader: 'svg-inline'
+      },
+      {
+        test: /\.scss$/,
```

Comment:
Not currently using sass in webpack - have to ignore to avoid errors, just a temporary ignore until the styling as a whole is addressed (maybe css-modules)